src/api.js

```diff
@@ -2,8 +2,8 @@ import fetch from 'isomorphic-fetch';
 
 import LedgerApi from './LedgerApi';
 
-export const API_HOST = 'https://data.detroitledger.org';
-//export const API_HOST = 'https://localhost:3000';
+//export const API_HOST = 'https://data.detroitledger.org';
+export const API_HOST = 'https://localhost:3000';
```

Comment:
oops don't commit this :)

Reply:
Maybe we should use a `.env` in this project